src/zimscraperlib/constants.py

```diff
@@ -20,39 +18,3 @@
 
 # list of mimetypes we consider articles using it should default to FRONT_ARTICLE
 FRONT_ARTICLE_MIMETYPES = ["text/html"]
-
-# list of mandatory meta tags of the zim file.
-MANDATORY_ZIM_METADATA_KEYS = [
```

Comment:
I don't think this is wise. I understand you want to keep metadata-related stuff together but the point of the constants module is for the developer to act as a config file. If some are spread into different places, then the existence of this file is in jeopardy.
Either we think it makes sense as it previously was or we harmonize and move other constants out as well.

Reply:
Got it